selftests/functional/test_interrupt.py

```diff
@@ -121,6 +121,9 @@ def wait_until_no_badtest():
         # Make sure the Killing test subprocess message did appear
         self.assertIn('Killing test subprocess', proc.get_output())
 
+    @unittest.skipIf(int(os.environ.get("AVOCADO_CHECK_LEVEL", 0)) < 2,
+                     "Skipping test that take a long time to run, are "
+                     "resource intensive or time sensitve")
```

Comment:
Btw did you consider adding the `skipIf` decorator to `avocado.core` or `avocado.utils`? I think it'd make sense, something like `@selftests.skipLevel(1)` which would produce `Skipping test under level 1...`